ntbs-service/Pages/Shared/_AlertsAndActions.cshtml

```diff
@@ -66,11 +66,13 @@
 
                 @if (Model.Notification.NotificationStatus != NotificationStatus.Draft)
                 {
-                    <print-button inline-template>
-                        <button nhs-button-type="Standard" classes="ntbsuk-button--secondary hidden" v-on:click="print" id="print-notification-overview-button">
-                            Print notification overview
-                        </button>
-                    </print-button>
+                    <form method="post">
+                        <print-button inline-template>
+                            <a role="button" draggable="false" class="nhsuk-button ntbsuk-button--secondary hidden" v-on:click="print" id="print-notification-overview-button">
+                                Print notification overview
+                            </a>
+                        </print-button>
+                    </form>
```

Comment:
I don't think the fake form way of doing this is a good one - semantically this isn't a form so this is misleading, and could potentially hurt us in terms of accessibility too. This [page](https://www.learnrazorpages.com/security/request-verification) mentions several alternatives - did you try those? Using `@Html.AntiForgeryToken()` seems simpler.

Reply:
Yes that's much easier, I did see that page when I was hunting for a way to do it but I don't think I read far enough down! I have added the anti forgery token and also that means I can revert the button back from an <a> tag.

Reply:
Excellent :-)